src/pybind/mgr/dashboard/frontend/src/app/shared/components/grafana/grafana.component.ts

```diff
@@ -19,6 +19,7 @@ export class GrafanaComponent implements OnInit, OnChanges {
   panelStyle: any;
   grafanaExist = false;
   mode = '&kiosk';
+  datasource = 'Dashboard1';
```

Comment:
Just curious, why `Dashboard1` instead just `Dashboard`?

Reply:
`Dashboard` was the name of the old datasource set by Ceph-ansible in Nautilus/4.x. We could make it configurable too, but given this doesn't change so often, I don't think it makes sense now.

Reply:
I know it's too late for this observation, but `Dashboard1` is a misleading name as it does not relate to any of the [grafana datasources](https://grafana.com/docs/grafana/latest/datasources/#supported-data-sources) (in this case we use [prometheus](https://github.com/ceph/ceph/tree/master/monitoring/prometheus)).